pyomo/core/tests/transform/test_strip_bounds.py

```diff
@@ -0,0 +1,93 @@
+"""Tests stripping of variable bounds."""
```

Comment:
I think you should add a test for the logic at the beginning of the transformation where you check if the instance already has the component maps.